src/App.jsx

```diff
@@ -14,57 +15,63 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [value, setValue] = useState('Jam');
```

Comment:
`value` is not the best naming for this constant: it doesn't let us understand its meaning. I'd advise to use `selectedGood` name, as suggested in the description, or something like that.